packages/grafana-ui/src/components/ConfirmButton/DeleteButton.tsx

```diff
@@ -13,16 +13,19 @@ export interface Props {
   /** Disable button click action */
   disabled?: boolean;
   'aria-label'?: string;
+  /** Close after delete button is clicked */
```

Comment:
Nit: redundant docs - it's just proxy to already documented property (in ConfirmButton.tsx)

Reply:
yes, but you still need to add it here to get the description in - for example - vscode